package.json

```diff
@@ -3,8 +3,11 @@
   "version": "5.0.0",
   "private": true,
   "dependencies": {
-    "@emotion/react": "latest",
```

Comment:
Why are you removing these dependencies? We need `emotion` for MUI to work properly

Reply:
Added "@emotion/react": "latest"